packages/components/src/core/InputCheckbox/index.tsx

```diff
@@ -67,6 +67,7 @@ const InputCheckbox = (props: CheckboxProps): JSX.Element => {
       caption={caption}
       control={
         <StyledCheckbox
+          tabIndex={0}
```

Comment:
I'm surprised we need this, because inputs are already focusable.

If we remove this, does anything bad happen? I suspect it'll still work without it.

Reply:
Thanks, it still works